tests/openvino/test_quantization.py

```diff
@@ -1085,7 +1085,7 @@ def preprocess_function(examples, tokenizer):
 
 
 class OVTrainerTest(unittest.TestCase):
-    SUPPORTED_ARCHITECTURES_WITH_EXPECTED_QUANTIZED_MATMULS = (("albert", 63, 39),)
```

Comment:
This change is due to the fact that this test is somewhat non-deterministic, please see ticket 142945